.env

```diff
@@ -0,0 +1 @@
+REDIS_URL=redis://127.0.0.1/1:6062
```

Comment:
Que penses-tu d'ajouter un commentaire comme quoi le document n'est pas encrypté et qu'il faut pas ajouter de secret ?

C'est clairement overkill, mais bon, habituellement les .env sont encryptés et on le prend pour acquis

Reply:
On pourrait aussi l'encrypter par défaut comme les autres, ce serait plus clean, right?

Reply:
Ouais, pour simplifier le tout on pourrait l'encrypter avec la key qui est déjà dans 1pass pour face-verification

Il faut ajouter le snippet suivant dans le fichier .gitattributes :
```
.env* filter=git-crypt diff=git-crypt
```

En ensuite lock le file avec la commande: 
`git-crypt lock /path/to/git-key`